android/app/src/main/AndroidManifest.xml

```diff
@@ -26,18 +26,22 @@
             </intent-filter>
         </activity>
         <activity
-            android:name="labs.pooh.eaterslab.ui.activity.map.MapSearchActivity"
-            android:label="@string/search_map_title"/>
+            android:name=".ui.activity.map.MapSearchActivity"
+            android:label="@string/search_title" />
         <activity
-            android:name="labs.pooh.eaterslab.ui.activity.main.MainActivity"
+            android:name=".ui.activity.main.MainActivity"
             android:hardwareAccelerated="true"
-            android:label="@string/app_name"/>
+            android:label="@string/app_name" />
         <activity
-            android:name="labs.pooh.eaterslab.ui.activity.settings.SettingsActivity"
-            android:label="@string/title_activity_settings"/>
+            android:name=".ui.activity.settings.SettingsActivity"
+            android:label="@string/title_activity_settings" />
         <activity
-            android:name="labs.pooh.eaterslab.ui.activity.loading.LoadingConnectionActivity"
-            android:launchMode="singleTop"/>
+        android:name=".ui.activity.search.SearchActivity"
+        android:label="@string/search_title"/>
+        <activity
+            android:name=".ui.activity.loading.LoadingConnectionActivity"
+            android:launchMode="singleTop" />
+
     </application>
 
-</manifest>
+</manifest>
```

Comment:
try to rebase your git branch on master to keep these things the same (they might have change in some time, I'm sorry for this but let's try to fix this by rebase process)